config/engines.yml

```diff
@@ -25,6 +25,12 @@ duplication:
   community: false
   enable_regexps:
   default_ratings_paths:
+  default_config:
```

Comment:
Not a blocker for this, but this file's getting kind of hairy. At some point somebody should probably break this up into individual YAML files in a `config/engines` directory or something.